docs/secret-rotation.md

```diff
@@ -0,0 +1,254 @@
+# Secret Rotations &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; **FluxCD**
+
+This brief guide should outline all the steps needed to be able to swap out a set of secrets or credentials with FluxCD GitOps.
+
+## [**Pre-requisites**](#pre-requisites-link)
+#### [Useful Resources](#useful-resources-link)
+#### [Applications & Software](#applications--software-link)
+
+## [**Outline**](#outline-link)
+#### [Cluster Maintenance](#cluster-maintenance-link)
+&nbsp; &nbsp; &nbsp; &nbsp; - Basics of what is done in a standard secret rotation.
```

Comment:
Please remove all instances of `&nbsp;`. Markdown should use tabs for indentation where necessary (and necessary usually only means to create sub-lists inside lists).

Reply:
Done